packages/grid/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -224,6 +224,11 @@ export interface DataGridPropsWithDefaultValues {
    * @default "client"
    */
   filterMode: GridFeatureMode;
+  /**
+   * The milliseconds delay to wait after a keystroke before triggering filtering.
+   * @default 50
+   */
+  filterDebounceMs: number;
```

Comment:
Should I reuse it for the quickfilter? Should I add an equivalent prop for it?

Reply:
Ok, done. I'm wondering if we shouldn't rename it to `debounceMs` and use it for all our input debounce values, we also need something like that here:

https://github.com/mui/mui-x/blob/01f92ac6d0bbea6c523278e8fae5e6153e78b42d/packages/grid/x-data-grid/src/components/cell/GridEditInputCell.tsx#L74

Reply:
I'm not sure about this, you might want to use different values there. For example, with server-side filtering, you might want to increase the debounce time for filtering, without impacting the editing debounce which is done on the client side